style.css

```diff
@@ -435,11 +436,431 @@ hr {
   display: flex;
   gap: 20px;
   justify-content: center;
-  margin-bottom: 30px;
+  margin-top: 20px;
 }
 
-.footer-hr {
-  width: 50%;
-  margin: auto;
-  border: 3px solid #000;
+.project-cards {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  padding: 10px;
+  grid-column-gap: 10px;
+  grid-row-gap: 10px;
+}
+
+.card1 {
+  background-image: url(./img/Icons/first.png);
+  height: 500px;
+  background-repeat: no-repeat;
+  background-size: 100%;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  vertical-align: bottom;
+}
+
+.card2 {
+  background-image: url(./img/Icons/second.png);
+  height: 500px;
+  background-repeat: no-repeat;
+  background-size: 100%;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  vertical-align: bottom;
+}
+
+.card3 {
+  background-image: url(./img/Icons/third.png);
+  height: 500px;
+  background-repeat: no-repeat;
+  background-size: 100%;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  vertical-align: bottom;
+}
+
+.card4 {
+  background-image: url(./img/Icons/fourth.png);
+  height: 500px;
+  background-repeat: no-repeat;
+  background-size: 100%;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  vertical-align: bottom;
+}
+
+.chips {
+  padding: 0 0;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  position: absolute;
+  bottom: 0;
+}
+
+.chips h3 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  padding-left: 10px;
+}
+
+.chips p {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  padding-left: 10px;
+}
+
+.card-ul {
+  display: flex;
+  flex-direction: row;
+  gap: 25px;
+  justify-content: center;
+}
+
+.card1 button {
+  height: 50px;
+  background-color: #f55800;
+  color: #fff;
+  border: none;
+  font-size: 17px;
+}
+
+.card1 button:active {
+  background-color: #e05304;
+  color: #f4f1f0;
+}
+
+.card1 button:hover {
+  background-color: #fff5e1;
+  color: #f55800;
+}
+
+.hidden {
+  visibility: hidden;
+}
+
+@media screen and (max-width: 768px) {
+  #fullwrap {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  #fullwrap > * {
+    order: 999;
+  }
+
+  #header {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    position: fixed;
```

Comment:
- [ ] Please check The [Video ](https://www.youtube.com/watch?v=mI7jbfxGGVw)  I recorded 😉 ,
I provided some great feedback I hope this will help you very much